public/assets/stylesheets/main.css

```diff
@@ -547,4 +547,209 @@ body {
     .DSC-logo{
         width: 90%;
     }
-}
\ No newline at end of file
+}
+
+/******************************************** MODAL ***********************************************/
+
+/* default for 320,1024,1440px */
+.modal-cont {
+	top: 0;
+	right: 0;
+	position: fixed;
+	height: 100%;
+	z-index:9999999;
+	font-family: google sans;
+}
+
+.modal-hold {
+	width: 320px;
+	height: 900px;
```

Comment:
change the height to 100vh and add overflow-y: scroll for browsers with shorter heights ( to scroll through the remaining contents )

Reply:
applied changes